src/GraphQL/Resolvers/NameFieldResolver.cs

```diff
@@ -92,5 +93,7 @@ Expression CreateAccessorExpression()
                 }
             }
         }
+
+        private static readonly PropertyInfo _sourcePropertyInfo = typeof(IResolveFieldContext).GetProperty(nameof(IResolveFieldContext.Source));
```

Comment:
Repeated across codebase many times.

Reply:
Yeah, I was thinking that also.  If you have a suggestion of where to store it as an `internal` property or field, I can consolidate.  `IResolveFieldContext` is an interface, so I can't add it there.  Really shouldn't be more than a half dozen instances however, so I don't think it's a real big deal.

Reply:
Not a real big deal. May be internal property in `AutoregisteringHelper`.